sx/pisa3/pisa_util.py

```diff
@@ -510,7 +510,7 @@ def __init__(self, uri, basepath=None):
             if len(urlParts[0]) > 1 :
 
                 # External data
-                if basepath:
+                if basepath and not (uri.startswith("http://") or uri.startswith("https://")):
```

Comment:
You can use `uri.startswith(('http://', 'https://'),)`, e.g. pass a tuple instead.  Make sure to test it.